frontend/javascripts/oxalis/model/volumetracing/volumelayer.js

```diff
@@ -408,11 +433,17 @@ class VolumeLayer {
     let p2;
     for (let i = 0; i < contourList.length; i++) {
       p1 = this.get2DCoordinate(contourList[i]);
-
+      p2 = this.get2DCoordinate(contourList[(i + 1) % contourList.length]);
```

Comment:
With this change, brush circles will be closed automatically now (in the same way the trace tool closes the contour in the end), because for the last value of `i`, `p1` will be the last entry of the `contourList` and `p2` will be the first entry of the `contourList` (see the modulo). However in the brushing case, we don't want to do that, we only want to fill the structure if it was closed by the user, not auto-close it.
I would suggest to check whether `i !== contourList.length - 1` in line 445 and only call the `fillRectangle` function if that's the case and add a comment explaining why.